cards/src/main/resources/cards/custom/group0/other_death_knight/msg/minion_icebox_snowbuddy.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Icebox Snowbuddy",
+  "baseManaCost": 5,
+  "type": "MINION",
+  "heroClass": "SPIRIT",
+  "baseAttack": 3,
+  "baseHp": 5,
+  "rarity": "RARE",
+  "description": "Battlecry: Deal damage equal to this minion's Attack randomly split among all enemies.",
+  "battlecry": {
+    "spell": {
+      "class": "MissilesSpell",
+      "target": "ENEMY_CHARACTERS",
```

Comment:
Needs `"ignoreSpellDamage": true` maybe?? I'm not sure.

Reply:
Prolly not, just copied from C'Thun